homeassistant/components/doorbird/entity.py

```diff
@@ -0,0 +1,38 @@
+"""The Tesla Powerwall integration base entity."""
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import Entity
+
+from .const import (
+    DOMAIN,
+    DOORBIRD_INFO_KEY_BUILD_NUMBER,
+    DOORBIRD_INFO_KEY_DEVICE_TYPE,
+    DOORBIRD_INFO_KEY_FIRMWARE,
+    MANUFACTURER,
+)
+from .util import get_mac_address_from_doorstation_info
+
+
+class DoorBirdEntity(Entity):
+    """Base class for doorbird entities."""
+
+    def __init__(self, doorstation, doorstation_info):
+        """Initialize the entity."""
+        super().__init__()
+        self._doorstation_info = doorstation_info
+        self._doorstation = doorstation
+        self._mac_addr = get_mac_address_from_doorstation_info(doorstation_info)
+
+    @property
+    def device_info(self):
+        """Doorbird device info."""
+        firmware = self._doorstation_info[DOORBIRD_INFO_KEY_FIRMWARE]
+        firmware_build = self._doorstation_info[DOORBIRD_INFO_KEY_BUILD_NUMBER]
+        return {
+            "identifiers": {(DOMAIN, self._mac_addr)},
```

Comment:
Identifiers and connections behave the same way, except that connections also tell something about where the identifier is used. No need to duplicate them. Can't hurt either.

Reply:
Good to know

removed `identifiers`